### PR TITLE
Global Page: Bug when there is no houses in the DB

### PR DESCRIPTION
**Pull request by ronyyosef**
Error when viewing the Global page when there are no houses in the DB

### CODE REVIEW

**Review by ShellyGolden — APPROVED** [review]

**Review by Yoavsc129 — APPROVED** [review]

File: house/views.py [hunk 1]
```diff
@@ -16,19 +16,20 @@ def home_page(request):
...
             houses = _filter_houses_by_form(cleaned_data, houses)
-    else:
-        form = HouseForm()
 
     categories_amounts = Expenses.average_expenses_of_houses_by_categories(houses)
     context = {
         'all_houses': houses,
-        "categories": [category.get('category') for category in categories_amounts],
-        "amounts": [amount.get('average') for amount in categories_amounts],
+        "categories": [category.get('category') for category in categories_amounts.all()],
```
Comment (LiorKGOW) [inline house/views.py:31]: Why did you add the .all() ? Shouldn't it be just the Public Houses?

File: tests/global_page_tests.py [hunk 2]
```diff
@@ -23,3 +23,8 @@ def test_global_doesnt_show_private_houses(self, db, client):
         houses_in_global = render.context["all_houses"]
         assert houses_in_global.filter(public=True).count() == 1
         assert houses_in_global.filter(public=False).count() == 0, "Private houses are shown"
+
+    def test_global_page_no_houses(self, db, client):
+        response = client.get('/global/')
+        assert len(response.context["all_houses"]) == 0, "There are houses in global page"
```
[the remark this answers is not in the document]
Comment (ronyyosef) [reply tests/global_page_tests.py:29]: github copilot haha, will fix it

[the remark this answers is not in the document]
Comment (LiorKGOW) [reply tests/global_page_tests.py]: Nice 😆

**Review by LiorKGOW — APPROVED** [review]

**Comment by shlior7** [comment]
I haven't seen any bug or problem when there are no houses, to check that i copied the new tests you created in this PR without the fix and they still passed. also when Im at the main branch and there are no houses this is what you see 
image

which is ok and only problem is we need to change **person** to **Houses** in `no person in the database`, which you can do in this PR with keeping the tests checking this case for future changes but the change to views is unnecessary

**Comment by ronyyosef** [comment]
> I haven't seen any bug or problem when there are no houses, to check that i copied the new tests you created in this PR without the fix and they still passed. also when Im at the main branch and there are no houses this is what you see image
> 
> which is ok and only problem is we need to change **person** to **Houses** in `no person in the database`, which you can do in this PR with keeping the tests checking this case for future changes but the change to views is unnecessary

The bug : 

I didn't find an exact way to test this specific bug
I add a test to make sure that the view function is running successfully
